.eslintrc.json

```diff
@@ -28,38 +16,27 @@
     "no-shadow": "off",
     "react/require-default-props": "off",
     "jsx-a11y/control-has-associated-label": "off",
+    "jsx-a11y/label-has-associated-control": "off",
```

Comment:
why was this lint option disabled?

Reply:
I think it's a byproduct of how the EventDetailsFormItem component is structured, we have
```tsx
<label htmlFor____/>
<EventDetailsFormItem>
  <input id=____/>
</EventDetailsFormItem>
```
but for some reason the ESLint rule complains so we're silencing it.

Reply:
so it looks like our rules require BOTH an `htmlFor` and nesting the input inside the label, we can make the rule `"jsx-a11y/label-has-associated-control": [2, { "assert": "either" }],` to allow just one https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md